assets/m4/mysql_release.m4

```diff
@@ -19,7 +19,7 @@ AC_DEFUN([mysql_AC_HAVE_MYSQL_VERSION],
 			 		MYSQL_MICRO=`echo $VERS | cut -d '.' -f 3`
 					MARIA=`echo $MYSQL_MICRO | cut -d '-' -f 2`
 
-					if [[ "$MARIA" == *aria* ]]; then
+					if [[[ "$MARIA" == *aria* ]]]; then
```

Comment:
I assume this line will output: `if [[ "$MARIA" == *aria* ]]; then`, correct?
If yes, it will not work as `[[ ]]` is a bash builtin. `configure` script should be POSIX compatible unless you explicitly run it via bash (default shebang is `#! /bin/sh`).

I did a small test:
```bash
# test-mysql.sh
VERS=`mysql --version 2>&1 | cut -d ' ' -f 6 | cut -d ',' -f 1`
MYSQL_MICRO=`echo $VERS | cut -d '.' -f 3`
MARIA=`echo $MYSQL_MICRO | cut -d '-' -f 2`
if [[ "$MARIA" == *aria* ]]; then
  echo "Maria DB"
else
  echo "NOT Maria DB"
fi
```
```bash
$ /bin/sh test-mysql.sh
test-mysql.sh: 5: test-mysql.sh: [[: not found
NOT Maria DB
```

Maybe you tested this on a system where `sh` is symlinked to `bash`? On my Ubuntu 18.04 `sh` points to `dash`.

This should work:
`if test -z ${MARIA##*aria*}; then`

Or replace with switch:
```bash
case "$MARIA" in
  *aria*)
    echo "Maria DB"
    ;;
  *)
    echo "NOT Maria DB"
    ;;
esac
```

Reply:
+1 for the switch solution

Reply:
> I assume this line will output: `if [[ "$MARIA" == *aria* ]]; then`, correct?

Yes
> 
> Maybe you tested this on a system where `sh` is symlinked to `bash`? On my Ubuntu 18.04 `sh` points to `dash`.

I was probably on a system where `sh` is symlinked to `bash` indeed.
switch solution sounds fine to me too then.